app/src/js/middleware/request.js

```diff
@@ -84,22 +99,21 @@ export const requestMiddleware = ({ dispatch, getState }) => (next) => (action)
       })
       .catch((error) => {
         if (error.response) {
-          const { data, status } = error.response;
+          const { status } = error.response;
           return handleError(
             {
               id,
               type,
-              error: data,
+              error,
               requestAction,
               statusCode: status
             },
             next
           );
         }
-        handleError({ id, type, error, requestAction }, next);
+        return handleError({ id, type, error, requestAction }, next);
```

Comment:
I'm guessing this was never used? I think that's an error that happens in axios. idk but this is probably what was intended.